### DISCUSSION TITLE
Any know how add kapersky scan to context menu?

### DISCUSSION THREAD

**Discussion by xRorkr** [moudey/Shell#205] [Q&A]
every time I activate the contextual menu I disable the option to scan with kapersky, does anyone know how to add it? my level of understanding or modification of this program is almost minimal, please, if you can help me.

**Comment by hiranokite** [comment]
Have you tried "More options" on context menu?

**Reply by xRorkr** [reply]
yes, but not..

**Comment by moudey** [comment]
@xRorkr Please clarify the issue

**Reply by xRorkr** [reply]
I don't see the option to send a scan from the context menu, I have kapersky installed.

**Reply by moudey** [reply]
Does it appear if Shell is disabled?

**Reply by xRorkr** [reply]
Yes ! but it is not always there, it only appears when I send files or usb drives to be analyzed, otherwise it is not visible, but with the context menu it is never there...

**Reply by hiranokite** [reply]
> Yes ! but it is not always there, it only appears when I send files or usb drives to be analyzed, otherwise it is not visible, but with the context menu it is never there...

Can you send a image of the default windows menu with these Kapersky menus?

**Comment by xRorkr** [comment]
sorry for not answering before, my windows is in spanish no problem?
